### PR TITLE
Iavp merge

### PR DESCRIPTION
**Pull request by Jrad7**

### CODE REVIEW

File: pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm [hunk 1]
```diff
@@ -280,43 +290,53 @@ sub _suggest_imunify360 {
 sub _suggest_iav {
     my ($self) = @_;
 
-    if ( !$self->{iav}{installed} && !$self->{iav}{licensed} ) {
+    if ( !$self->{iav}{installed} && !$self->{iavp}{licensed} ) {
```
Comment (atoomic) [inline pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm]: we are inside `_suggest_iav ` should not we only use `$self->{iav}` and not `iavp` here?

File: pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm [hunk 2]
```diff
@@ -280,43 +290,53 @@ sub _suggest_imunify360 {
...
         $self->_avplus_advice( action => 'installav', advice => 'bad' );
-        $self->_avplus_advice( action => 'upgrade',   advice => 'info' );
     }
-    elsif ( !$self->{iav}{installed} && $self->{iav}{licensed} ) {
-        $self->_avplus_advice( action => 'installplus', advice => 'bad' );
+    elsif ( $self->{iav}{installed} ) {
+
+        require Cpanel::RPM;
```
Comment (atoomic) [inline pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm]: maybe could use `_can_load_module` also here?

File: pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm [hunk 3]
```diff
@@ -326,7 +346,7 @@ sub _upgrade_avplus_text {
 
 sub _install_av_text {
     my ($self) = @_;
-    my $install_av_url = $self->base_path('scripts13/install_imunifyav_SECURITYADVISOR');
+    my $install_av_url = $self->base_path('scripts14/install_imunifyav_SECURITYADVISOR');
```
Comment (atoomic) [inline pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm]: should not this use `_get_script_number` to use the accurate script depending on cPanel&WHM version?

File: pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm [hunk 4]
```diff
@@ -336,7 +356,7 @@ sub _install_av_text {
 
 sub _install_avplus_text {
     my ($self) = @_;
-    my $install_plus_url = $self->base_path('scripts13/install_imunifyavplus_SECURITYADVISOR');
+    my $install_plus_url = $self->base_path('scripts14/install_imunifyavplus_SECURITYADVISOR');
```
Comment (atoomic) [inline pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm]: same comment here about using `_get_script_number`

File: pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm (hunk 1 above)
Comment (Jrad7) [reply pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm]: They are related to each other. We want to suggest IAV install if IAV is not installed and IAVP is not licensed.

If IAVP is licensed we ask them to install IAVP in the _suggest_iavp() function.

File: pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm (hunk 2 above)
Comment (Jrad7) [reply pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm]: indeed

File: pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm (hunk 3 above)
Comment (Jrad7) [reply pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm]: For IAV and IAVP there is only one script# to use.

_get_script_number is for Imunify360 only

File: pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm (hunk 1 above)
Comment (atoomic) [reply pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm]: thanks

File: pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm (hunk 3 above)
Comment (atoomic) [reply pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm]: perfect

**Pull request merged**